backend/apps/volontulo/forms.py

```diff
@@ -119,6 +119,52 @@ class Meta(object):
             'volunteers_limit',
         ]
 
+    def clean(self):
+        super(CreateOfferForm, self).clean()
+        started_at = self.cleaned_data.get('started_at')
+        finished_at = self.cleaned_data.get('finished_at')
+        if started_at and finished_at:
+            if started_at > finished_at:
+                self.add_error(
+                    'started_at',
+                    """Data rozpoczęcia akcji nie może być późniejsza,
```

Comment:
Please refactor all messages into class variables. It will avoid repeating text (easy to make a typo this way!).